webpack.config.js

```diff
@@ -202,6 +202,11 @@ module.exports = {
           ]
         ),
         use: ['null-loader'],
+      },
+      {
+        test: /\.js$/,
```

Comment:
N.B. the English locale does not live in `moment/locale` for some reason (presumably American exceptionalism)

Reply:
Out of curiosity, any reason why you use null-loader for this instead of webpack's [IgnorePlugin](https://webpack.js.org/plugins/ignore-plugin/)?

Reply:
lol good call, ignoring Moment locales is literally the example use

Reply:
@ajgreenb ok so I played around with IgnorePlugin and I ran into require errors when using it to remove unneeded parts of the `stylelint` library, which we currently do with the null loader. Given that the two seem to be roughly a wash in terms of interface/structure, I’m inclined to just keep using the null loader.